Dynavity/Dynavity/view/ToolbarView.swift

```diff
@@ -51,6 +55,17 @@ struct ToolbarView: View {
         HStack {
             Spacer()
             addButton
+            // SwiftUI does not allow attaching multiple sheets to the same element.
```

Comment:
Would it be better to use something like https://stackoverflow.com/questions/58837007/multiple-sheetispresented-doesnt-work-in-swiftui

i.e. use an enum to represent the currently displayed sheet. The enum can also contain the required arguments for the imagepicker (aka sourceType). 

Reason is because I'm thinking the current implementation may be susceptible to the buggy situation where multiple sheets are presented concurrently if state is poorly handled (which I'm assuming is non-ideal, at least for now)